### PR TITLE
bug report issue templated created

### PR DESCRIPTION
**Pull request by harshmathurx**
Bug report issue template added as mentioned in issue #129 
More issue templates can be added later by other contributors

### CODE REVIEW

**Comment by harshmathurx** [comment]
hey, @kaiwalyakoparkar can you please review and mention any changes if required?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+name: "🐛 Bug Report"
+about: Report a reproducible bug or regression.
+title: 'Bug: '
+labels: 'type: bug'
```
Comment (kaiwalyakoparkar) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Hey I guess we don't have `type: bug` label so could you please change to one we have

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (harshmathurx) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: We do right? check the second type
image

**Comment by kaiwalyakoparkar** [comment]
@harshmathurx Please enable `issues` tab on your forked repo so that I can check this template :)

**Comment by harshmathurx** [comment]
> 
> 
> @harshmathurx Please enable `issues` tab on your forked repo so that I can check this template :)

just did

**Comment by kaiwalyakoparkar** [comment]
@harshmathurx Please make sure you get yourself issue assigned before making a PR. If same behavior is found next time then your PR will have to be closed instantly. Hope you understand :slightly_smiling_face:

**Comment by harshmathurx** [comment]
> @harshmathurx Please make sure you get yourself issue assigned before making a PR. If same behavior is found next time then your PR will have to be closed instantly. Hope you understand :slightly_smiling_face: 

I'm sorry, my bad 
I'll be mindful of that

**Comment by harshmathurx** [comment]
> 
> 
> @harshmathurx We are requesting for issue template in `.yml` and not in `.md` could you please change that?

I'm not well versed with .yml files so I'm kinda confused

**Pull request closed without merging**